config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
+
+HypervisorPath is an optional absolute path to the hypervisor binary used by VM-based runtimes.
+
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
```

Comment:
On Wed, Apr 27, 2016 at 05:19:49PM -0700, Lai Jiangshan wrote:
“`ImagePath` is the path of the image outside the vm”

Ah.  So it's something like:

  "ImagePath": "path/to/some-image"

where some-image is a tarball-like thing that includes the root
filesystem.  That sounds more widely applicable than the VM case.  For
example, it would be nice to be able to run Linux namespace/cgroup
containers with a tarred rootfs without having to unpack the tarball.
Why can't you have:

  "root": {
    "path": "path/to/some-image/my/root"
  }

and teach the VM runtime logic like:

* Nibble path components from root.path until you find a path that
  exists on the host filesystem.
* The existing path (‘path/to/some-image’ in the example above) is the
  image file (your ImagePath).
* The nibbled components (‘my/root’ in the example above) are the path
  to root inside the image (your root.path).

The fact that your filesystem switches from Btrfs (or some such) to
OVA (or some such) at path/to/some-image doesn't seem like information
that requires a VM-specific setting to convey.